### ISSUE TITLE
[TSINGMICRO] some op test failed without --ref cpu

### ISSUE THREAD

**Issue by tsingmicro-public-e** [FlagOpen/FlagGems#4131]
### Describe the bug

[TSINGMICRO] some op test failed without --ref cpu

**Comment by tengqm** [comment]
This issue description is too simple. If this is meant to be an umbrella issue, please mark it as an umbrella so that we can track it.

**Comment by tengqm** [comment]
## Non-working operators on TsingMicro backend

Based on PR #3782, the following **174 operators** are marked as not working (`skipif`) on the TsingMicro backend:

- `acos`
- `adaptive_avg_pool2d`
- `add`
- `addcdiv`
- `addcdiv_`
- `addcmul`
- `addcmul_`
- `all`
- `allclose`
- `amax`
- `aminmax`
- `angle`
- `any`
- `arange`
- `arctanh`
- `argmax`
- `argsort`
- `atan`
- `atan2`
- `atanh`
- `atanh_`
- `avg_pool2d`
- `baddbmm`
- `bernoulli`
- `bmm`
- `cat`
- `cauchy`
- `celu`
- `clamp`
- `clamp_min`
- `col2im`
- `concatenate`
- `constant_pad_nd`
- `conv_transpose1d`
- `conv1d`
- `conv2d`
- `conv3d`
- `copy`
- `copysign`
- `diff`
- `digamma`
- `div`
- `dot`
- `dropout`
- `elu`
- `eq`
- `equal`
- `erf`
- `exponential`
- `feature_dropout`
- `fix`
- `flash_mla`
- `flip`
- `floor_divide`
- `fmin`
- `fmod`
- `fmod_`
- `fused_add_rms_norm`
- `gather`
- `ge`
- `gelu_and_mul`
- `geometric`
- `glu`
- `greater`
- `grid_sample`
- `hardswish`
- `histc`
- `hypot`
- `i0`
- `index`
- `index_add`
- `index_copy`
- `index_put`
- `index_reduce`
- `isclose`
- `isfinite`
- `isin`
- `kron`
- `le`
- `leaky_relu`
- `lerp`
- `linspace`
- `log_softmax`
- `logaddexp`
- `logical_and`
- `logical_or`
- `logical_xor`
- `logical_xor_`
- `logit`
- `logspace`
- `masked_fill`
- `masked_select`
- `max`
- `max_pool2d`
- `mhc`
- `min`
- `minimum`
- `mm`
- `mode`
- `multinomial`
- `nonzero`
- `nonzero_numpy`
- `normal`
- `pad`
- `poisson`
- `polar`
- `pow`
- `prelu`
- `prod`
- `rand`
- `rand_like`
- `randint`
- `randint_like`
- `randn`
- `randn_like`
- `randperm`
- `reciprocal`
- `remainder`
- `renorm`
- `repeat`
- `repeat_interleave`
- `replication_pad3d`
- `roll`
- `round`
- `rrelu_with_noise_backward`
- `rsqrt`
- `safe_softmax`
- `scaled_dot_product_attention`
- `scatter_add`
- `scatter_reduce`
- `scatter_reduce_two`
- `scatter_src`
- `searchsorted`
- `segment_reduce`
- `sigmoid`
- `signbit`
- `silu`
- `silu_and_mul`
- `silu_and_mul_with_clamp`
- `skip_layer_norm`
- `slice_backward`
- `smooth_l1_loss`
- `soft_margin_loss`
- `softmax`
- `softplus`
- `sparse_mla_fwd_interface`
- `special_chebyshev_polynomial_v`
- `special_i0e`
- `special_i1`
- `stack`
- `std`
- `sub`
- `tan`
- `tanh`
- `threshold`
- `to_copy`
- `tril`
- `triu`
- `trunc_`
- `uniform`
- `unique`
- `upsample_bicubic2d`
- `upsample_bicubic2d_aa`
- `upsample_linear1d`
- `upsample_nearest_exact1d`
- `upsample_nearest1d`
- `upsample_nearest2d`
- `upsample_nearest3d`
- `upsample_trilinear3d`
- `var`
- `var_mean`
- `vstack`
- `weight_norm`
- `weight_norm_interface`
